pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -631,7 +631,7 @@ func GetOpenshiftBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 			Rules: []authorizationapi.PolicyRule{
-				authorizationapi.NewRule("get", "list", "watch", "update").Groups(kapiGroup).Resources("replicationcontrollers").RuleOrDie(),
+				authorizationapi.NewRule("get", "list", "watch", "update", "delete").Groups(kapiGroup).Resources("replicationcontrollers").RuleOrDie(),
```

Comment:
https://github.com/openshift/origin/blob/master/pkg/cmd/server/bootstrappolicy/controller_policy.go#L67 seems to be fixed in HEAD